src/schemapack/_internals/export/mermaid.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
```

Comment:
I've restructured the module structure accordingly. Just as a background, some application make a distinction between "save"/"load" and "export"/"import", where the former is for the application native format and the latter is for non-native format(s) which might be lossy or only importable or exportable. Thus I thought this might actually provide a proper distinction.

Reply:
OK, I was not aware of that. Makes also sense. At least once we have multiple export targets.